src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -232,8 +232,8 @@ private function checkResourceLocator($resourceLocator, $resourceLocatorPrefix)
      */
     protected function getRedirectRoute(Request $request, $url)
     {
-        $requestFormat = $request->getRequestFormat();
-        $format = 'html' !== $requestFormat ? '.' . $requestFormat : '';
+        $requestFormat = $request->getRequestFormat(null);
+        $formatSuffix = $requestFormat ? '.' . $requestFormat : '';
```

Comment:
@nnatter I'm not sure with your change this will add `.html` and we should never redirect to a `.html` url as that should always be stripped.

Reply:
no i tested that - `$request->getRequestFormat(null)` will return `null` if the url does not contain any format.

even if you access an url with `.html` in the end (eg `http://sulu.localhost:8000/new-test.html`), the `RedirectExceptionSubscriber` will remove that before this method is called 🙂

Reply:
okay never saw the `RedirectExceptionSubscriber` 🙈